pages/cart/index.js

```diff
@@ -0,0 +1,218 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import { toast } from "react-toastify";
+
+import Header from "../../components/Header";
+import Footer from "../../components/Footer";
+import Divider from "../../styles/UIKit/Divider";
+import UIBUtton from "../../styles/UIKit/Button";
+import useApi from "../../utils/useApi";
+import CartConstants from "../../config/CartConfig";
+
+const Cart = () => {
+  const api = useApi();
+  const [cartItems, setCartItems] = useState([]);
+  const [cartData, setCartData] = useState({});
+
+  const fetchData = async () => {
+    const { data } = await api.get("cart");
+    const {
+      data: { items },
+    } = data;
+
+    setCartItems(items);
+    setCartData(data.data);
+  };
+
+  const handleDeleteCartItem = (itemId) => {
+    deleteCartItem(itemId);
+  };
+
+  const deleteCartItem = async (itemId) => {
+    toast.info("Deleting Cart item");
+    const newCartItems = cartItems.filter(
+      (item) => item.product._id !== itemId
+    );
+
+    setCartItems(newCartItems);
+
+    try {
+      await api.delete(`cart/product/${itemId}`);
+      fetchData();
+
+      toast.success("Deleted cart item successfully");
+    } catch (err) {
+      toast.error("Some error has occurred");
+      fetchData();
+    }
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  return (
+    <div>
+      <Header />
+      <CartContainer>
+        <CartTitleContainer>
+          <FlexAdjuster>
+            <EmptyBox />
+          </FlexAdjuster>
+          <FlexAdjuster>
+            <EmptyBox />
+          </FlexAdjuster>
+          <FlexAdjuster>
+            {" "}
+            <CartHeading> {CartConstants.header.product} </CartHeading>
+          </FlexAdjuster>
+          <FlexAdjuster>
+            <CartHeading> {CartConstants.header.price} </CartHeading>
+          </FlexAdjuster>
+          <FlexAdjuster>
+            <CartHeading> {CartConstants.header.quantity} </CartHeading>
+          </FlexAdjuster>
+          <FlexAdjuster>
+            <CartHeading> {CartConstants.header.total} </CartHeading>
+          </FlexAdjuster>
+        </CartTitleContainer>
+        <CartDivider />
+        {cartItems.map((cartItem) => (
+          <div key={cartItem._id}>
+            <CartContent>
+              <FlexAdjuster>
+                <CloseX
```

Comment:
You can use an Icon from the icons library here, and also adjust the width of this column, it's too wide

Reply:
this has been resolved